index.html

```diff
@@ -46,6 +46,9 @@ <h1>Visual regression testing for web apps.</h1>
         <p class="getItBlock col-10">
           <span class="cta"><a href="https://github.com/garris/BackstopJS" class="github-link"><img src="assets/github-icon.png" class="github-icon" alt="Get it on GitHub"></a></span>
           <span class="cta"><a href="https://github.com/garris/BackstopJS">Get it on github!</a></span>
+          <a href="https://github.com/garris/BackstopJS" class="puppy-link">
```

Comment:
Is this going to show up on the backstop brochure page?   It's a bit of a sequitur 🤔